python/mxnet/contrib/onnx/_import/import_onnx.py

```diff
@@ -126,13 +130,22 @@ def from_onnx(self, graph):
                     len(node.output), len(mxnet_sym.list_outputs()), op_name))
             for k, i in zip(list(node.output), range(len(node.output))):
                 self._nodes[k] = mxnet_sym[i]
+
+            # splitting params into args and aux params
+            for args in mxnet_sym.list_arguments():
+                if args in self._params:
+                    argDict.update({args: nd.array(self._params[args])})
+            for aux in mxnet_sym.list_auxiliary_states():
+                if aux in self._params:
```

Comment:
Same comment as above in line 136

Reply:
We could get rid of "if" check here, but this works as safety net , will not lead to incorrect accesses.